### DISCUSSION TITLE
CTkSlider: font size

### DISCUSSION THREAD

**Discussion by BatelG** [TomSchimansky/CustomTkinter#955] [General]
How can I resize the "from", "to" labels from CTkSlider?

**Comment by TomSchimansky** [comment]
I don't know what you mean? The slider has no labels.

**Comment by BatelG** [comment]
The slider shows the "from" and "to" values, but it is very small. I was
working on 3.11 version until yesterday and it was ok. Yesterday I have
upgraded to 5.0.3 version and it was changed.
